extensions/vscode/src/api/resources/Configurations.ts

```diff
@@ -20,38 +20,48 @@ export class Configurations {
   // 200 - success
   // 404 - not found
   // 500 - internal server error
-  get(configName: string) {
+  get(configName: string, params?: { dir?: string }) {
     const encodedName = encodeURIComponent(configName);
     return this.client.get<Configuration | ConfigurationError>(
       `/configurations/${encodedName}`,
+      { params },
```

Comment:
Do we need to encode the param dir value, before sending it, in ALL of the API requests? (Asking for this one only, but asking for all API requests).

Reply:
Yes we do. Fantastic catch. I've been testing, but didn't test with a double nested one. I will do that now, and update this PR.

Reply:
Did some digging into this and testing locally. Axios encodes params passed using its `AxiosRequestConfig` for us. It doesn't encode the the URL which is why we need it for names.

https://github.com/axios/axios/issues/463#issuecomment-250937871